cumulusci/tasks/sample_data/capture_sample_data.py

```diff
@@ -19,7 +20,15 @@ class CaptureSampleData(BaseSalesforceApiTask):
             )
         },
         "extraction_definition": {
-            "description": "A file describing what to be extracted."
+            "description": "A file describing what to be extracted. "
+            "Defaults to `datasets/{datasetname}/{datasetname}.extract.yml` if it exists."
+        },
+        "loading_rules": {
+            "description": (
+                "Path to .load.yml file containing rules to use when loading the mapping. "
+                "Defaults to`datasets/{datasetname}/{datasetname}.load.yml` if it exists. "
+                "Multiple files can be comma separated."
```

Comment:
Is this line accurate? I don't see any indication we load multiple loading rule files.